flow-model-generator/modules/flow-model-index-generator/src/main/resources/create-index.sql

```diff
@@ -22,6 +22,7 @@ CREATE TABLE Function (
     description TEXT,
     package_id INTEGER,
     return_type JSON, -- JSON type for return type information
+    resource_path TEXT NOT NULL,
```

Comment:
There can entries such as `FUNCTION` and `REMOTE` which does not require this column.

Reply:
Your points are valid. I added this change only for now. Later anyway we need to keep a separate table to handle resource paths. For the private beta lets keep this way. @hasithaa wdyt?

Reply:
Since this is an internal change let's keep this for now, once the release is done let's refactor.